mirdata/annotations.py

```diff
@@ -79,6 +79,26 @@ def __repr__(self):
         return repr_str
 
 
+class MultiAnnotator(Annotation):
```

Comment:
nit: does it make sense for this to be a subclass of `Annotation`? I think it might be better to just subclass `object`

Reply:
But semantically it is an annotation no? Plus it prints so nicely <3

Reply:
we might want to print multi-annotations differently than Annotations though, no?